zerberus/sfz.cpp

```diff
@@ -24,6 +24,7 @@
 #include "zone.h"
 #include "sample.h"
 #include "zerberus.h"
+#include "zerberusgui.h"
```

Comment:
I'm not a big fan of this include. It would be better to have sfz and the ui decoupled. Best way would be to do like we do for progress `zerberus->setLoadProgress`and so add a wascanceled book in zerberus class.